shipLHC/AdvMuFilterHit.h

```diff
@@ -15,6 +15,7 @@ class AdvMuFilterHit : public SndlhcHit
 
     // Constructor from MuFilterPoint
     AdvMuFilterHit(Int_t detID, const std::vector<AdvMuFilterPoint*>&);
```

Comment:
`AdvMuFilterHit(Int_t detID, const std::vector<AdvMuFilterPoint*>&);`
this is no longer needed if we have the new version with const pointers